### PR TITLE
Create react onepager

### PR DESCRIPTION
**Pull request by joebau0815**
Ich habe angefangen die Seite neu zu entwickeln. Diesmal auf Basis von React

### CODE REVIEW

File: new-website/debug.log [hunk 1]
```diff
@@ -238,3 +238,18 @@
 [0725/182729.279:ERROR:crash_report_database_win.cc(469)] failed to stat report
 [0725/182729.279:ERROR:crash_report_database_win.cc(469)] failed to stat report
 [0725/182729.280:ERROR:crash_report_database_win.cc(469)] failed to stat report
+[0725/183950.798:ERROR:crash_report_database_win.cc(469)] failed to stat report
```
Comment (use-to) [inline new-website/debug.log]: Delete this whole file! Dont commit log files in general...

File: .gitignore [hunk 2]
```diff
@@ -1 +1,23 @@
-note
\ No newline at end of file
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (use-to) [inline .gitignore]: Add your logfiles here

File: Dockerfile [hunk 3]
```diff
@@ -1,6 +0,0 @@
-
-FROM nginx:alpine
-MAINTAINER cryptic-game
```
Comment (use-to) [inline Dockerfile]: Is there a reason for this deletion?

File: debug.log [hunk 4]
```diff
@@ -0,0 +1,315 @@
+[0724/220336.242:ERROR:crash_report_database_win.cc(469)] failed to stat report
```
Comment (use-to) [inline debug.log]: No logs!

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <title>React App</title>
```
Comment (use-to) [inline public/index.html]: Change the title to `Cryptic`

File: public/manifest.json [hunk 6]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```
Comment (use-to) [inline public/manifest.json]: Update the naming of that project...

File: Dockerfile (hunk 3 above)
Comment (joebau0815) [reply Dockerfile]: Thought it's unneccessary, because it belongs to the old side

Comment (use-to) [reply Dockerfile]: No this is necessary for deployment

File: server/app.py [hunk 7]
```diff
@@ -0,0 +1,9 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route("/")
+def index():
+    return "Hello World!"
```
Comment (use-to) [inline server/app.py]: This seems to me that the app isnt ready yet...

Comment (joebau0815) [reply server/app.py]: This will become the CMS, but the current app is using hard coded content

Comment (joebau0815) [reply server/app.py]: So it is ready

File: Dockerfile (hunk 3 above)
Comment (joebau0815) [reply Dockerfile]: Then you have to add it to the new app, I don't know how this works

Comment (use-to) [reply Dockerfile]: Okay, we will just ignore this for a moment :+1:

**Pull request merged**